Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gliderlabs/alpine
+
+COPY requirements.txt /opt/elodie/requirements.txt
+COPY docs/requirements.txt /opt/elodie/docs/requirements.txt
+COPY elodie/tests/requirements.txt /opt/elodie/elodie/tests/requirements.txt
+WORKDIR /opt/elodie
+
+RUN apk add --update boost-python ca-certificates exiftool exiv2 make python py-pip && \
```

Comment:
It would be call to split a bit this long RUN command to better leverage docker's caching ability.

At least 3 stages would be good:
1. dependencies
2. compilation
3. installation

That way if something fails during compilation - which is I would presume the likeliest thing to happen - only that step will be redone.
This means also the COPY commands should not be add the top but between steps 1 and 2.